content/en/users/getting-started/architecture/_index.md

```diff
@@ -33,33 +34,34 @@ research.
 
 ## Federated IaaS
 
-The EGI Federated Cloud Infrastructure as a Service (IaaS) resource centres
+The EGI Federated Cloud Infrastructure-as-a-Service (IaaS) resource centres
 deploy a Cloud Management Framework (CMF) that provide users with an API-based
 service for management of Virtual Machines and associated Block Storage to
 enable persistence and Networks to enable connectivity of the Virtual Machines
 among themselves and third party resources.
 
-The IaaS federation is a thin layer that brings the providers together with:
+The IaaS federation is a [thin layer](../../../internal) that brings the providers
+together with:
 
-- federated authentication;
-- resource discovery;
-- central VM image catalogue;
-- usage accounting; and
-- monitoring.
+- Federated authentication
+- Resource discovery
+- Central VM image catalogue
+- Usage accounting
+- Monitoring
 
 The IaaS capabilities (VM, block storage, network management) must be provided
 via community agreed APIs (OpenStack and/or OCCI are supported at the moment)
 that allow integration with EGI Check-in for authentication and authorisation of
 users. Those providers that limit the interaction to web dashboards and do not
 expose APIs to direct consumption for users cannot be considered part of the EGI
-IaaS Cloud Compute service.
+IaaS Cloud services.
 
 The information system provides a real-time view about the actual capabilities
-of federation participants. The EGI Configuration Database (GOCDB) contains the
-list of resource centres and their entry endpoints. Information about these
-endpoints is expressed in a standard format (GlueSchema 2.1) and pushed to
-consumers via the Argo Messaging System. The AppDB Information System collects
-this information in a central service for discovery.
+of federation participants. The EGI [Configuration Database](https://goc.egi.eu/)
```

Comment:
Here you could also link to the internal service documentation, it may be more meaningful for them (especially as they may not easily have access to the Configuration Database)